### PR TITLE
add EDFilter for selecting events w/in a given (intime) PU range

### PR DESCRIPTION
**Pull request by mtosi**
the title says all
this is a simple EDFilter meant for selecting events w/in a given (intime) PU range

### CODE REVIEW

**Comment by mtosi** [comment]
ciao,
@smuzaffar @monttj are there any issues w/ this PR ?
thanks

**Comment by monttj** [comment]
please test

**Comment by monttj** [comment]
+1

File: PhysicsTools/Utilities/plugins/PileUpFilter.cc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+#include "FWCore/Utilities/interface/StreamID.h"
+
+
+#include "SimDataFormats/PileupSummaryInfo/interface/PileupSummaryInfo.h"
+ 
+//
+// class declaration
+//
+
+class PileUpFilter : public edm::stream::EDFilter<> {
```
Comment (Dr15Jones) [inline PhysicsTools/Utilities/plugins/PileUpFilter.cc]: It looks like this could easily have been a `global::EDFilter<>` instead.

**Comment by mtosi** [comment]
@Dr15Jones thanks for the suggestion !

**Comment by monttj** [comment]
@cmsbuild please test

**Pull request merged**